pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -294,11 +294,11 @@ type Controller struct {
 	// taints instead of evicting Pods itself.
 	useTaintBasedEvictions bool
 
-	// if set to true, NodeController will taint Nodes based on its condition for 'NetworkUnavailable',
-	// 'MemoryPressure', 'PIDPressure' and 'DiskPressure'.
-	taintNodeByCondition bool
-
 	nodeUpdateQueue workqueue.Interface
+
+	// if set to true, Controller will skip taint nodes by condition.
+	// NOTE: used in scheduler integration test.
```

Comment:
please also add a comment on when should we remove that field :)

Reply:
done